docs/plugins.md

```diff
@@ -31,7 +31,16 @@ When plugins cannot be found automatically, you can load them with:
   });
   ```
 
-Prettier expects each of `pluginSearchDirs` to contain `node_modules` subdirectory, where `@prettier/plugin-*`, `@*/prettier-plugin-*` and `prettier-plugin-*` will be searched. For instance, this can be your project directory or the location of global npm modules.
+- Or the [Configuration File](configuration.md):
```

Comment:
We are not using `Or` in API section, should we remove `Or` here?

Reply:
Yeah, looks better without the `Or`.